config_builder.go

```diff
@@ -587,18 +587,13 @@ func (d *DefaultWalletImpl) BuildWalletConfig(ctx context.Context,
 		MigrateWatchOnly: d.migrateWatchOnly,
 	}
 
-	// Parse coin selection strategy.
-	switch d.cfg.CoinSelectionStrategy {
-	case "largest":
-		walletConfig.CoinSelectionStrategy = wallet.CoinSelectionLargest
-
-	case "random":
-		walletConfig.CoinSelectionStrategy = wallet.CoinSelectionRandom
-
-	default:
-		return nil, nil, nil, fmt.Errorf("unknown coin selection "+
-			"strategy %v", d.cfg.CoinSelectionStrategy)
+	coinSelectionStrategy, err := lnrpc.MapCoinSelectionStrategy(
+		d.cfg.CoinSelectionStrategy,
+	)
+	if err != nil {
+		return nil, nil, nil, err
 	}
+	walletConfig.CoinSelectionStrategy = coinSelectionStrategy
```

Comment:
I think we can leave this as is. We only want to do this conversion from config string to `wallet.CoinSelectionStrategy` _once_. It won't ever change. Then everywhere else where we need to know the global config, we can use this already parsed `wallet.CoinSelectionStrategy` value instead of re-parsing the string.

Reply:
sp perhaps all that is needed here is to make the `wallet.CoinSelectionStrategy` strategy more easily accessible from the rpcserver level